cli/script/command-executor.ts

```diff
@@ -1347,6 +1365,47 @@ function serializeConnectionInfo(accessKey: string, preserveAccessKeyOnLogout: b
     log(`\r\nSuccessfully logged-in. Your session file was written to ${chalk.cyan(configFilePath)}. You can run the ${chalk.cyan("code-push logout")} command at any time to delete this file and terminate your session.\r\n`);
 }
 
+function sessionList(command: cli.ISessionListCommand): Promise<void> {
+    throwForInvalidOutputFormat(command.format);
+
+    return sdk.getAccessKeys()
```

Comment:
The most tangible value is that if `getAccessKeys` returns everything, it becomes obvious to the developer that `removeAccessKey`, `editAccessKey` should work with whatever was returned from `getAccessKeys`. 

Unless we are saying that `removeAccessKey` and `editAccessKey` shouldn't work with session keys (and the fact that they in fact actually do today is not something developers is not an officially supported feature and is subject to change), then I will be happy to make this change